### PR TITLE
Rework advice on Text Wrapping

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

**Comment by fwilhe** [comment]
@mojavelinux Could not find a source for `Neo4j recommendation`. I would like to link to that. Do you have that handy?

File: docs/asciidoc-recommended-practices.adoc [hunk 1]
```diff
@@ -25,12 +25,18 @@ NOTE: I recommend _against_ using the +.asc+ extension. The connection to AsciiD
...
-Don't do it!
+Don't wrap the text at a certain amount of characters.
+The recommended way is to write one sentence per line.
 
-Sentence per line? (Neo4j recommendation)
+//todo refer to Neo4j recommendation
 
 AsciiDoc doesn't treat a wrapped line break as a hard line break.
 
+The reader won't notice how you handle line breaks, if the document is rendered to html/epub/pdf.
```
Comment (mojavelinux) [inline docs/asciidoc-recommended-practices.adoc]: I think this is better written as:

"The line breaks between contiguous lines will not be visible in the rendered document (i.e., as the reader sees it)."

**Comment by mojavelinux** [comment]
They seem to have gotten rid of that section in the latest guide. Here's the section from a previous version of their guide (still available online).

http://neo4j.com/docs/2.2.8/community-docs.html#_writing

**Comment by fwilhe** [comment]
Thanks for your comments, I updated the PR.

**Comment by mojavelinux** [comment]
Merged as 3289c80cdff98b2821dc74c4c0da3626f4d5b1bd with additional revisions in d42b9a9d736ff9ab3dc1253f922c05b8989b7d71.

As it turns out, Buckminster Fuller may have discovered this idea even before the age of computers. So this idea goes back pretty far (1930s). I documented that history in my revision. I also itemized some of the advantages.

**Comment by mojavelinux** [comment]
I also renamed the section to "One Sentence Per Line" since that is the focus.

See http://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

**Comment by fwilhe** [comment]
Nice, so I close this PR

**Comment by mojavelinux** [comment]
:+1:

**Comment by jaredmorgs** [comment]
This way of structuring sentences is really changing the way I write all my
Asciidoctor files. Great back story here as well.

On Thu, Mar 3, 2016, 18:22 Dan Allen <notifications@github.com> wrote:

> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/pull/522#issuecomment-191649058>
> .
>
-- 

Sent from Mobile.

**Comment by mojavelinux** [comment]
That's great to hear, @jaredmorgs. This technique is definitely more than just a style change. It unlocks something in the brain.

**Pull request closed without merging**